class.jetpack-cli.php

```diff
@@ -118,7 +118,7 @@ public function test_connection( $args, $assoc_args ) {
 		}
 
 		$body = wp_remote_retrieve_body( $response );
-		if ( ! $body ) {
+		if ( ! $body || is_wp_error( $body ) ) {
```

Comment:
This probably should add a new response conditional, as the error message being passed back (empty response body) wouldn't be accurate.